app/views/reports/housed_addresses/index.haml

```diff
@@ -0,0 +1,31 @@
+- title = _('Client Addresses')
+- content_for :title, title
+
+= render 'reports/crumbs_operational'
+%h1.mb-5= title
+
+= render 'filter'
+
+- if @addresses.present?
+  .d-flex
+    %p= page_entries_info @addresses
+    %p= paginate @addresses
+  .c-card.c-card--flush
+    .c-card__content.p-0
+      %table.table.table-striped
+        %thead
+          %tr
+            - headers.each do |header|
+              %th= header
+        %tbody
+          - @addresses.pluck(columns).each do |row|
+            %tr
+              - row.each do |value|
+                - if !!value == value # Check for boolean
+                  %td= value ? 'Y' : 'N'
```

Comment:
Let's use one of the various available methods in `application_helper`  Maybe, `yes_no`.